tests/shared_nfsv4/shared_nfsv4_test.go

```diff
@@ -0,0 +1,79 @@
+package tests
```

Comment:
I recently realized that adding new folders here is causing addition of new static test binaries each of which is 100MB. This increases the torpedo container image size.

Let's just add this shared_nfsv4_test.go file under the existing `basic` directory.

Reply:
thanks for letting me know, I will do that then. One concern is that, if there are some jenkins job that set up to run all basic tests, they will be broken, because this test would only succeed with the app sv4-svc-nfsv4

Reply:
They won't break. Since the jobs have to include this test in the FOCUS_TESTS to actually run the test. Just adding to the _test.go file will not run the test.

Reply:
good to know, will change then.